src/dom.js

```diff
@@ -0,0 +1,296 @@
+    createLocationHeader(`${data.city}, ${data.country}`);
+    return data;
+  };
+
+  const createSearchButton = () => {
+    selectors.searchButton = document.createElement('button');
+    selectors.searchButton.setAttribute('id', 'search');
+    selectors.contentSection.prepend(selectors.searchButton);
+  };
+
+  const ipLookUpAsyncOperations = (promise) => promise()
+    .then(data => {
+      createSearchButton();
+      return data;
+    })
+    .then(data => storeLocationAtHeader(data));
+
+  const createBackgroundDiv = () => {
+    const bg = document.createElement('div');
+    bg.setAttribute('id', 'bg');
+    selectors.body.appendChild(bg);
+  };
+
+  const createWeatherSection = () => {
+    selectors.weatherSection = document.createElement('section');
+    selectors.weatherSection.setAttribute('id', 'weather-stats');
+    selectors.contentSection.prepend(selectors.weatherSection);
+  };
+
+  const createWeatherTempHeader = (data) => {
+    selectors.weatherTempHeaderDiv = document.createElement('div');
+    selectors.weatherTempHeader = document.createElement('h1');
+    selectors.weatherTempHeaderDiv.setAttribute('id', 'weather-header');
+    selectors.weatherTempHeader.appendChild(document.createTextNode(data));
+    selectors.weatherTempHeaderDiv.appendChild(selectors.weatherTempHeader);
+    selectors.weatherSection.appendChild(selectors.weatherTempHeaderDiv);
+  };
+
+  const createTempsBtnsDiv = () => {
+    selectors.tempsBtnsDiv = document.createElement('div');
+    selectors.tempsBtnsDiv.setAttribute('id', 'temps-btns');
+    selectors.weatherTempHeaderDiv.appendChild(selectors.tempsBtnsDiv);
+  };
+
+  const createCelsiusButton = () => {
+    selectors.celsiusButton = document.createElement('button');
+    selectors.celsiusButton.setAttribute('id', 'celsius-button');
+    selectors.celsiusButton.appendChild(document.createTextNode('℃'));
+    selectors.tempsBtnsDiv.appendChild(selectors.celsiusButton);
+  };
+
+  const createFahrenheitButton = () => {
+    selectors.fahrenheitButton = document.createElement('button');
+    selectors.fahrenheitButton.setAttribute('id', 'fahrenheit-button');
+    selectors.fahrenheitButton.appendChild(document.createTextNode('℉'));
+    selectors.tempsBtnsDiv.appendChild(selectors.fahrenheitButton);
+  };
+
+  const storeWeathertemp = (data) => {
+    createWeatherTempHeader(`${Math.round(data.main.temp - 273.15)} ℃`);
+    createTempsBtnsDiv();
+    createCelsiusButton();
+    createFahrenheitButton();
+    return data;
+  };
+
+  const rerenderTempsStats = (temp, feelsLike, convertionOffset, tempSymbol) => {
+    selectors.weatherTempHeader.textContent = '';
+    selectors.weatherTempFeelsLike.textContent = '';
+    selectors.weatherTempHeader.appendChild(document.createTextNode(`${Math.round(temp - convertionOffset)} ${tempSymbol}`));
+    selectors.weatherTempFeelsLike.appendChild(document.createTextNode(`Feels Like: ${Math.round(feelsLike - convertionOffset)} ${tempSymbol}`));
+  };
```

Comment:
Like I said in previous review
The application makes different requests for the Celsius data and the Fahrenheit data otherwise it beats the point of the assignment.

See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/requests_requirement.png) from Pathwright that lists this as a requirement